source/features/pr-approvals-count.css

```
@@ -17,3 +17,7 @@
 .js-issue-row .color-text-secondary [aria-label*='changes'] {
 	color: var(--github-red) !important;
 }
+
+[aria-label*='approval']:after {
```

Let's match the existing selector and add a comment

```suggestion
/* Disable now-duplicate tooltip */
.js-issue-row .color-text-secondary [aria-label*='approval']:after {
```